tools/tuf/download-artifacts/download-artifacts.go

```diff
@@ -368,9 +370,10 @@ func osquerydCommand() *cli.Command {
 		},
 		Action: func(c *cli.Context) error {
 			return downloadComponents("generate-osqueryd-targets.yml", gitBranch, map[string]string{
-				"macos":   "osqueryd.app.tar.gz",
-				"linux":   "osqueryd",
-				"windows": "osqueryd.exe",
+				"macos":       "osqueryd.app.tar.gz",
+				"linux":       "osqueryd",
+				"linux-arm64": "osqueryd-arm64",
+				"windows":     "osqueryd.exe",
```

Comment:
We need to update the other two invocations of `downloadComponents` (for `orbit` and `Fleet Desktop`).

Reply:
Ah I'm realizing I didn't make a `goreleaser-linux-arm64` file, I missed that part of the release process